### PR TITLE
Added Navbar component to home page

### PR DESCRIPTION
**Pull request by JustinLohKingWei**
Created a navigation Bar component in a homepage. NavBar is created from a modified version of the Appbar Component, Includes Functioning Drop Down Pane . Also includes placeholder pages of the planning page.

# Description

1) Added NavBar component in a newly created components folder. Navbar displays component in a functional drop down bar. and also includes a non functional(for now) logout button
2) Added a home page as temporarily primary page. A placeholder for planning page is also created but not used .

## Link to work item

App bar documentation: https://material-ui.com/components/app-bar/

## How to Test

Changes are purely visual and can simply be tested by spinning up the app.

### CODE REVIEW

**Comment by obonobo** [comment]
Hey @JustinLohKingWei, your build is failing from the following error:

```plaintext
[INFO] src/index.js
[INFO]   Line 9:6:  'Home' is not defined  react/jsx-no-undef
```

**Comment by obonobo** [comment]
Also just an FYI, the CI is configured to be a bit more strict than a regular build on your local machine. It will fail on any warnings, so even if the build succeeds on your local machine (with warnings), those warnings will cause the CI to fail.

File: src/main/app/package.json [hunk 1]
```diff
@@ -5,10 +5,10 @@
   "proxy": "http://localhost:8080",
   "dependencies": {
     "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.0",
-    "bootstrap": "^4.6.0",
     "react": "^17.0.1",
     "react-bootstrap": "^1.4.3",
```
Comment (obonobo) [inline src/main/app/package.json:13]: @JustinLohKingWei can you also remove `react-bootstrap` as a dependency if we are going to be using mainly bootstrap

Comment (JustinLohKingWei) [reply src/main/app/package.json]: Working on it. Removed bootstrap via command

Comment (obonobo) [reply src/main/app/package.json]: Yes use: `yarn remove react-bootstrap`

Comment (JustinLohKingWei) [reply src/main/app/package.json]: Alright everything checks out.

**Review by obonobo — APPROVED** [review]
Looks great Justin!

**Pull request merged**